docs/source/migrating.rst

```diff
@@ -0,0 +1,189 @@
+Migrating from IPython
+======================
+
+`The Big Split <https://blog.jupyter.org/2015/04/15/the-big-split/>`__
+has changed a few things, moving the various language-agnostict
+components of IPython under the Jupyter umbrella. This doc should help
+you find what's changed, and how to keep doing what you've been doing.
+
+Where have all my files gone?
```

Comment:
I think this is a reactive document. Under normal circumstances, users don't have to do anything because migration is automatic. So I expect people coming here to be wondering why it's not already doing what they expect.